src/e2e-test/resources/pluginDataCyAttributes.properties

```diff
@@ -0,0 +1,23 @@
+## DATA-CY-ATTRIBUTES-DATAPLEX-PLUGIN-START
+dataplexProjectID=project
```

Comment:
most of the attributes are common in all the GCP plugins, it would be good to add the `dataplex` prefix only to attributes specific to dataplex and remove from others so that they can be reused in other plugins e2e-tests.